### PR TITLE
Implement Automatic Database Updates [GG-66]

### PR DESCRIPTION
**Pull request by kemptonk**
Implemented a custom django command for updating the web database and then created a script to set it to be automatically called.

### CODE REVIEW

**Comment by mraynor** [comment]
👍

File: GGProject/scripts/adminOptions.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 import ldap
 from django_auth_ldap.backend import LDAPBackend
+from GreeterGuru import settings
 
 
 def populate():
 
-    host = "192.168.200.128"		   # Server’s name/IP address
+    host = settings.SERVER_NAME		   # Server’s name/IP address
     username = "internship\\Administrator" # Any user’s login name on the server
```
Comment (lb-cake) [inline GGProject/scripts/adminOptions.py]: Does it make sense to have the username/ password be accessible from the "settings.py" similar to the above line?

File: GGProject/workflow/management/commands/sync_command.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+class Command(BaseCommand):
+
+    help = "Synchronizes web database employees with active directory users"
+    
+    def handle(self, *args, **kwargs):
+
+        host = settings.SERVER_NAME		   # Server’s name/IP address
+        username = "internship\\Administrator" # Any user’s login name on the server
```
Comment (lb-cake) [inline GGProject/workflow/management/commands/sync_command.py]: Same here. Does it make more sense that we can access this string from settings.py?

File: README.md [hunk 3]
```diff
@@ -53,6 +53,11 @@ $ python3 manage.py runserver
 $ rm -f tmp.db db.sqlite3
 ```
 
+## To enable auto-updates for the web database
+```bash
+$ bin\update.sh
```
Comment (lb-cake) [inline README.md]: Shouldn't this be a forward slash since we are developing for Unix systems?

**Review by lb-cake — CHANGES_REQUESTED** [review]
We talked about the issues. Everything else is good to go!

**Comment by kemptonk** [comment]
Everything should be good to go now

**Review by lb-cake — APPROVED** [review]

**Pull request merged**